scripts/Makefile

```diff
@@ -0,0 +1,19 @@
+#!make
+MAKEFLAGS += --silent
+
+# This allows us to accept extra arguments (by doing nothing when we get a job that doesn't match, 
+# rather than throwing an error).
+%: 
+    @: 
+
+# $(MAKECMDGOALS) is the list of "targets" spelled out on the command line
+stage: 
```

Comment:
Have we changed the target to stage?

Reply:
`~/Documents/myfork/docs-node(DOCSP-TESTLEG) $ make stage
You need a patch flag("commit" or "local") in your make command
~/Documents/myfork/docs-node(DOCSP-TESTLEG) $ make stage local
yoooo  upstream/master

diff --git a/source/index.txt b/source/index.txt
index 8f5e9f6..be2c765 100644
--- a/source/index.txt
+++ b/source/index.txt
@@ -6,7 +6,7 @@ MongoDB Node Driver
 
 .. include:: /includes/unicode-checkmark.rst
 
-The MongoDB Node driver documentation contains documentation specific to
+Hi hi hi The MongoDB Node driver documentation contains documentation specific to
 the Node driver. For the main MongoDB documentation, see the
 :manual:`MongoDB Manual </>`.
 
diff --git a/worker.sh b/worker.sh
index c615eb5..b8df9a6 100644
--- a/worker.sh
+++ b/worker.sh
@@ -1 +1 @@
-"build-and-stage-next-gen"
+"//build-and-stage-next-gen"

There was an error enqueing a staging job to docs autobuilder. Here is the error: MongoError: Cannot use a session that has ended
make: *** No rule to make target `local'.  Stop.`